omod/src/main/webapp/resources/htmlforms/vitals.xml

```diff
@@ -0,0 +1,329 @@
+<htmlform formUuid="a000cb34-9ec1-4344-a1c8-f692232f6edd"
+      formName="Vitals"
+      formEncounterType="68a71486-1a54-468f-ac3e-7091a9a79584"
+      formVersion="1.0"
+      formDescription="Vitals Form"
+      formAddMetadata="yes"
+      formUILocation="patientDashboard.visitActions"
+      formOrder="8"
+      formIcon="icon-medkit"
+      formShowIf="(visit.active || !visit.active)"
+      formDisplayStyle="Simple"
+      formLabel="Vitals Form">
+<style>
```

Comment:
@semujju Add indentation to all Javascript and CSS styles

Reply:
done

Reply:
Done